includes/class-edd-cli.php

```diff
@@ -1580,7 +1580,7 @@ public function download_logs( $args, $assoc_args ) {
 
 			$file_id_key = array_rand( $download_ids_with_files[ $product_id ], 1 );
 			$file_key    = $download_ids_with_files[ $product_id ][ $file_id_key ];
-			edd_add_file_download_log( array(
+			$log_id      = edd_add_file_download_log( array(
```

Comment:
```suggestion
			edd_add_file_download_log( array(
```

The ID is no longer needed.